src/transform/WidenImage.js

```diff
@@ -1,30 +1,99 @@
 import './WidenImage.css'
 import elementUtilities from './ElementUtilities'
 
+/**
+ * Gets array of ancestors of element which need widening.
+ * @param  {!HTMLElement} element
+ * @return {!Array.<HTMLElement>} Zero length array is returned if no elements should be widened.
+ */
+const ancestorsToWiden = element => {
+  const widenThese = []
+  let el = element
+  while (el.parentNode) {
+    el = el.parentNode
+    // No need to walk above 'content_block'.
+    if (el.classList.contains('content_block')) {
+      break
+    }
+    widenThese.push(el)
+  }
+  return widenThese
+}
+
+/**
+ * Sets style value.
+ * @param {HTMLElement.style} style
+ * @param {!string} key
+ * @param {*} value
+ * @return {void}
+ */
+const updateStyleValue = (style, key, value) => {
+  style[key] = value
+}
+
+/**
+ * Sets style value only if value for given key already exists.
+ * @param {HTMLElement.style} style
+ * @param {!string} key
+ * @param {*} value
+ * @return {void}
+ */
+const updateExistingStyleValue = (style, key, value) => {
+  const existingValue = style[key]
+  if (!existingValue) {
```

Comment:
Feel free to ignore but if you negate this conditional, you can move `updateStyleValue()` into it and omit the `return`.

Reply:
Ah good point. I've been doing more "early" returns to keep nesting shallower, but in this case it's only one line.

Reply:
Fixed!